application/src/main/java/org/opentripplanner/standalone/configure/LoadApplication.java

```diff
@@ -78,7 +81,8 @@ public ConstructApplication appConstruction() {
       DataImportIssueSummary.empty(),
       factory.emptyEmissionsDataModel(),
       factory.emptyStopConsolidationRepository(),
-      factory.emptyStreetLimitationParameters()
+      factory.emptyStreetLimitationParameters(),
+      new DefaultFareService()
```

Comment:
Make consistent with the rest.

Reply:
I realised that I actually don't want to do it, because that would mean adding an @Inject annotation to DefaultFareService, which I don't want to.